CMake/Util.cmake

```diff
@@ -8,13 +8,12 @@ macro (carla_declare_git_dependency NAME URL TAG)
     GIT_TAG ${TAG}
     GIT_PROGRESS ON
     GIT_SHALLOW ON
+    FIND_PACKAGE_ARGS
```

Comment:
I think this must go in the same line like:
```cmake
FIND_PACKAGE_ARGS NAMES ${NAME}
```

I'm no sure if can cause issues.